Docker dep fix: Add lib package 40 thing that clusterprofiler needs

cansavvy opened the pull request:
## Purpose:
Fix the precipitating event of #312 - clusterprofiler needed a dependency to install even though it docker didn't stop building when it failed to install. 

This was the problem: 
```
#12 608.2 Error in dyn.load(file, DLLpath = DLLpath, ...) : 
#12 608.2   unable to load shared object '/usr/local/lib/R/site-library/igraph/libs/igraph.so':
#12 608.2   libglpk.so.40: cannot open shared object file: No such file or directory
#12 608.2 Calls: <Anonymous> ... asNamespace -> loadNamespace -> library.dynam -> dyn.load
#12 608.2 Execution halted
#12 608.2 ERROR: lazy loading failed for package ‘enrichplot’
#12 608.2 * removing ‘/usr/local/lib/R/site-library/enrichplot’

ERROR: dependency ‘enrichplot’ is not available for package ‘clusterProfiler’
```

## Strategy
Add back in `libglpk40` which appears to be the thing it needs.

jashapiro reviewed (commented):
Do we want to add `options(warn=2)` in this PR as suggested:  

https://github.com/AlexsLemonade/refinebio-examples/issues/312#issuecomment-713586320

cansavvy commented:
> Do we want to add `options(warn=2)` in this PR as suggested:
> 
> [#312 (comment)](https://github.com/AlexsLemonade/refinebio-examples/issues/312#issuecomment-713586320)
 
Seems like a good time for that, sure! Will add now.

jashapiro reviewed (approved):
Confirming that this docker image does build as expected.

cansavvy commented:
Yay. `option(warn=2)` made it fail like it should in https://github.com/AlexsLemonade/refinebio-examples/runs/1288528019?check_suite_focus=true (I deleted the linux dependency that gave the warnings in the first place). 

Now I've reverted that test in 46835b5 so it should pass again.
